package.json

```diff
@@ -25,6 +25,7 @@
     "postinstall": "husky install"
   },
   "dependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
```

Comment:
RE: to the above tho, we should move to using next to package stuff and api to package stuff themselves asap cuz maintaining babel is a PITA